Source/EasyNetQ.Hosepipe/ErrorRetry.cs

```diff
@@ -37,24 +38,17 @@ public void RepublishError(Error error, QueueParameters parameters)
             {
                 try
                 {
-                    if (error.Exchange != string.Empty)
-                    {
-                        model.ExchangeDeclarePassive(error.Exchange);
-                    }
-
                     var properties = model.CreateBasicProperties();
                     error.BasicProperties.CopyTo(properties);
-
                     var body = errorMessageSerializer.Deserialize(error.Message);
-
-                    model.BasicPublish(error.Exchange, error.RoutingKey, true, properties, body);
+                    model.BasicPublish("", error.Queue, true, properties, body);
```

Comment:
@kzhen I hope it is enough to cover your request.

Reply:
Brilliant, thanks!

Reply:
I hope I will be able to complete this PR today.